compiler/src/iree/compiler/Dialect/LinalgExt/IR/AggregatedOpInterfaceImpl.cpp

```diff
@@ -274,7 +275,11 @@ static Value computeSubAndExp2(OpBuilder &builder, Location loc,
         Value in = convertScalarToDtype(b, loc, args[0], args[1].getType(),
                                         /*isUnsignedCast=*/false);
         Value diff = b.create<arith::SubFOp>(loc, args[1], in);
-        Value weight = b.create<math::Exp2Op>(loc, diff);
+        Value weight;
+        if (useExp2)
+          weight = b.create<math::Exp2Op>(loc, diff);
+        else
+          weight = b.create<math::ExpOp>(loc, diff);
```

Comment:
Can you split the exp2 addition into a seperate patch? It's a good addition.

Reply:
Sure, thanks!